src/components/MoneyTemporaryForRefactorRequestConfirmationList.js

```diff
@@ -317,8 +317,8 @@ function MoneyTemporaryForRefactorRequestConfirmationList({
     const isMerchantEmpty = !iouMerchant || iouMerchant === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT;
     const isMerchantRequired = isPolicyExpenseChat && !isScanRequest && shouldShowMerchant;
 
-    const isCategoryRequired = canUseViolations && Boolean(policy.requiresCategory);
-    const isTagRequired = canUseViolations && Boolean(policy.requiresTag);
+    const isCategoryRequired = canUseViolations && lodashGet(policy, 'requiresCategory', false);
+    const isTagRequired = canUseViolations && lodashGet(policy, 'requiresTag', false);
```

Comment:
Just wondering, didn't we hope not to have to use lodashGet anymore or am i crazy? can't we do this instead? Or is this ugly to y'all?

```suggestion
    const isCategoryRequired = canUseViolations && (policy?.requiresCategory ?? false);
    const isTagRequired = canUseViolations && (policy?.requiresTag ?? false);
```

Reply:
Till recently, this triggered the linter in the .js files. Has that changed?

Reply:
I accepted the suggestion, I'll revert it if linter complains.